launcher/net/PasteUpload.cpp

```diff
@@ -59,7 +59,7 @@ void PasteUpload::downloadFinished()
         m_reply.reset();
         return;
     }
-    else if (statusCode != 200)
+    else if (statusCode != 200 && statusCode != 201)
```

Comment:
What is the purpose of this

Reply:
pacebin (and maybe some other pastebins) return 201 Created upon successful paste creation.